src/rlx_config.erl

```diff
@@ -129,8 +129,11 @@ load({extended_start_script, ExtendedStartScript}, {ok, State}) ->
     {ok, rlx_state:extended_start_script(State, ExtendedStartScript)};
 load({extended_start_script_extensions, ExtendedStartScriptExtensions}, {ok, State}) ->
     {ok, rlx_state:extended_start_script_extensions(State, ExtendedStartScriptExtensions)};
-load({start_script_type, StartScriptType}, {ok, State}) ->
-    {ok, rlx_state:start_script_type(State, StartScriptType)};
+load({include_start_scripts_for, IncludeStartScriptsFor}, {ok, State}) when is_list(IncludeStartScriptsFor) ->
+    {ok, rlx_state:include_start_scripts_for(State, IncludeStartScriptsFor)};
+load({start_script_type, "powershell"}, {ok, State}) ->
+    ?log_warn("Config option start_script_type is deprecated, use {include_start_scripts_for, \"powershell\"} instead", []),
```

Comment:
if it was only used in release candidates of relx you can deprecate it without worrying too much I guess?

Reply:
Oh, hm, didn't realize it was added in the rc's, but it looks like you are right.